index.html

```diff
@@ -87,29 +87,31 @@ <h1>Disitalisez votre cabinet<span class="red-punctuation">.</span>
                     Stock et vous transmet en toute securite
                     les pieces comptables de vas clients:</p>
             </div>
-            <div class="link">
-                <a class="icon-link" href="#">
-                    <i class="link-icon fas fa-laptop"></i>
-                    <span class="link-text">Collecte</span>
-                </a>
-            </div>
-            <div class="link">
-                <a class="icon-link" href="#">
-                    <i class="link-icon far fa-file-alt"></i>
-                    <span class="link-text">Avantages client</span>
-                </a>
-            </div>
-            <div class="link">
-                <a class="icon-link" href="#">
-                    <i class="link-icon far fa-clipboard"></i>
-                    <span class="link-text">Productivite</span>
-                </a>
-            </div>
-            <div class="link">
-                <a class="icon-link" href="#">
-                    <i class="link-icon far fa-handshake"></i>
-                    <span class="link-text">Collaboratif</span>
-                </a>
+            <div class="product-link">
+                <div class="link">
```

Comment:
`<div class="link">` is not very well named, since `link` has other implications in HTML

Reply:
Start using BEM

Reply:
Will rename